### DISCUSSION TITLE
How to suppress image inclusion in category descriptions?

### DISCUSSION THREAD

**Discussion by ddnexus** [retypeapp/retype#300] [Q&A] [answered]
Sometimes the image in the page is not a great addition to the description of the page added in the Categories list (e.g. when an image is very small for example, or if you just want to have text).

What is the official way to suppress the auto inclusion?

**Comment by geoffreymcgill** [comment, marked as answer]
Please set `image: null` in the page metadata. 

```
---
image: null
---
# Sample Page

This is a sample page. 
```

I will add this tip to the [`image`](https://retype.com/configuration/page/#image) config docs. 

Hope this helps.

**Reply by ddnexus** [reply]
Wow... 4 mins to get the answer! Thank you!

**Reply by geoffreymcgill** [reply]
Sorry, we will try to be more efficient in the future. 😉

**Comment by ddnexus** [comment]
@geoffreymcgill this is currently not working.

E.g. [page](https://github.com/benkoshy/pagy/blob/retype/docs/extras/bootstrap.md) and [categories](https://benkoshy.github.io/pagy/categories/frontend/)

**Reply by geoffreymcgill** [reply]
> this is currently not working.

@ddnexus Thanks for the links. I will investigate.
